pages/integrations/slack.md

```diff
@@ -25,6 +25,10 @@ Once you have granted access to your Slack workspace, give it a description, cho
 
 <%= image "buildkite-slack-connected.png", width: 1458/2, height: 1540/2, alt: "Screenshot of Buildkite Slack Notification Settings, requesting a description, your choice of text or emoji message themes, which pipelines and branches to include, and which build states should trigger a notification" %>
 
+>🚧
+> Default quota limit for number of slack notification services that can be added to an organization is 50. If you are an `Enterprise` customer and need higher quota limit, please reach out to support@buildkite.com.
```

Comment:
I'm so sorry we got to this so late @nsuma8989 !

I just have a quick suggestion here...

```suggestion
> 🚧
> The default quota limit for the number of Slack notification services that can be added to an organization is 50. If you are an Enterprise customer and need higher quota limit, please reach out to support@buildkite.com.
```